engines/dfc_provider/config/routes.rb

```diff
@@ -1,13 +1,24 @@
 # frozen_string_literal: true
 
 DfcProvider::Engine.routes.draw do
-  namespace :api do
-    scope :dfc_provider, as: :dfc_provider, path: '/dfc_provider' do
-      resources :enterprises, only: [:show] do
-        resources :catalog_items, only: [:index, :show]
-        resources :supplied_products, only: [:show]
+  constraints DfcProvider::ActivationConstraint.new do
+    namespace :admin do
+      resource :dfc_provider_settings, only: %i[show]
+    end
+
+    namespace :api do
+      namespace :v0 do
+        scope :dfc_provider, as: :dfc_provider, path: '/dfc_provider' do
+          # Just used for the JSON-LD context base url generation
+          get '/', to: redirect('/'), as: :base
```

Comment:
@mkllnk Still a bit hacky but better than adding an action to the abstract controller.

Reply:
Does it need `, as: :base`?

So we are just redirecting to the OFN home page, right?

Reply:
Actually, I just need it to generate a route, we will never access it. It should be the base used to access other resources with JSON-LD ids. We can think about something better later.

Reply:
So the result is we get a named route for `/api/v0/dfc_provider` called `api_v0_dfc_provider_base`, and it just redirects to the app root `/` (eg the home page)...?